File: tests/test-minecraft-osp/src/main/java/pize/tests/voxelgame/ServerMain.java
package pize.tests.voxelgame;

import pize.tests.voxelgame.server.Server;

public class ServerMain{
    
    public static void main(String[] args){
        final Server server = new Server(){
            @Override
            public void run(){
                super.run();
            }
        };
        
        server.run();
        
        while(!Thread.interrupted());
    }
    
}

File: tests/test-minecraft-osp/src/main/kotlin/pize/tests/voxelgame/ServerMain.kt
package pize.tests.voxelgame

import pize.tests.voxelgame.server.Server

object ServerMain {
    @JvmStatic
    fun main(args: Array<String>) {
        val server: Server = object : Server() {
            override fun run() {
                super.run()
            }
        }
        server.run()
        while (!Thread.interrupted());
    }
}
